src/server.ts

```diff
@@ -119,77 +144,86 @@ app.post("/killConnection", async (req, res) => {
     webParticipant &&
     req.body.participantId == webParticipant.id
   ) {
-    // TODO - move the null check inside of the functions
     console.log("deallocating all configured resources on exit");
     await killSipUriLeg(bridgeParticipant);
     await deleteParticipant(bridgeParticipant);
     await deleteParticipant(webParticipant);
     await deleteSession();
+    clientWs.close();
   }
 });
 
 /**
- * The browser will hit this endpoint to initiate a call to the outbound phone number
+ * handle an incoming call that has been placed to the TN that has a
+ * provisioned association with this application
+ * (long story - see setup in the README)
  */
-app.post("/callPhone", async (req, res) => {
-  console.log("calling a phone", req.body.calledTelephoneNumber);
-  const outboundPhoneNumber = req.body.calledTelephoneNumber;
-  if (
-    !outboundPhoneNumber ||
-    !outboundPhoneNumber.match(/^\+1[2-9][0-9]{9}$/)
-  ) {
-    console.log("missing or incorrectly formatted telephone number");
-    res
-      .status(400)
-      .send(
-        `missing or incorrectly formatted telephone number${outboundPhoneNumber}`
-      );
-  }
-  await callPhone(outboundPhoneNumber);
-  res.status(204).send();
-});
+app.post("/incomingCall", async (req, res) => {
+  console.log(`incoming call received from ${req.body.from}`);
+  updateCallStatus("inbound call");
 
-/**
- * the /bridgeCallAnswered api call completes the linkage of the webRTC and
- * V2 Voice environments
- */
-app.post("/bridgeCallAnswered", async (req, res) => {
-  const callId = req.body.callId;
-  console.log(
-    `received answered callback for bridging call ${callId} to ${req.body.to}`
-  );
+  // setup the interconnection to webRTC
+  bridgeParticipant = await createParticipant("hello-world-phone");
+  // TODO - confirm the async behavior on call setup
+  callSipUri(bridgeParticipant);
```

Comment:
callSipUri is async... You might want to put an await here

Reply:
that was intentionally async, but I put a comment in to ensure that was clear.